### ISSUE TITLE
Update library to support current version of Chai

### ISSUE THREAD

**Issue by paul-barry-kenzan** [prodatakey/karma-dirty-chai#4]
Hi @joshperry 
We discussed this issue within the comments of PR #1, but haven't seen the update applied. Any movement on this?

https://github.com/prodatakey/karma-dirty-chai/pull/1#issuecomment-182006363

Full message when running `npm install --save-dev karma-dirty-chai`

```
npm WARN peerDependencies The peer dependency chai@<1.10.0 || >1.10.0 <3 included from karma-dirty-chai will no
npm WARN peerDependencies longer be automatically installed to fulfill the peerDependency 
npm WARN peerDependencies in npm 3+. Your application will need to depend on it explicitly.
npm WARN peerDependencies The peer dependency dirty-chai@^1.2.0 included from karma-dirty-chai will no
npm WARN peerDependencies longer be automatically installed to fulfill the peerDependency 
npm WARN peerDependencies in npm 3+. Your application will need to depend on it explicitly.
npm ERR! Darwin 14.5.0
npm ERR! argv "node" "/usr/local/bin/npm" "install" "--save-dev" "karma-dirty-chai"
npm ERR! node v0.12.2
npm ERR! npm  v2.12.0
npm ERR! code EPEERINVALID

npm ERR! peerinvalid The package chai does not satisfy its siblings' peerDependencies requirements!
npm ERR! peerinvalid Peer dirty-chai@1.2.2 wants chai@<1.10.0 || >1.10.0 <4
npm ERR! peerinvalid Peer karma-chai@0.1.0 wants chai@*
npm ERR! peerinvalid Peer karma-dirty-chai@1.0.1 wants chai@<1.10.0 || >1.10.0 <3

npm ERR! Please include the following file with any support request:
npm ERR!     npm-debug.log
```

**Comment by paul-barry-kenzan** [comment]
@joshperry 
Any update on the Chai dependency?

**Comment by webOS101** [comment]
Ping.

**Comment by webOS101** [comment]
I created https://github.com/prodatakey/karma-dirty-chai/pull/5 to update to match the version requested by dirty-chai.  I didn't look at updating to more recent versions.

**Comment by joshperry** [comment]
Merged #5, I'll push soon. Apparently the Apple store wifi blocks SSH.
